conf/messages.en

```diff
@@ -115,14 +115,14 @@ whichBestDescribesYou.heading = Who is this application for?
 whichBestDescribesYou.businessOwner = The business or organisation that I work for or own
 whichBestDescribesYou.businessRepresentative = A different business or organisation that I am representing
 whichBestDescribesYou.checkYourAnswersLabel = This application is for
-whichBestDescribesYou.error.required = Select who this application is for
+whichBestDescribesYou.error.required = Select if this application is for the business or organisation you work for or own or a business, organisation or individual that you're representing
```

Comment:
wrong character ' . replace it for **you’re**

Reply:
Needs to be a smart apostrophe e.g. you’ll .  (Alt=Shift-} on Mac keyboard)